### PR TITLE
no bundle exec

### PR DESCRIPTION
**Pull request by gintama91**
lol idk if this is what noah meant there or something else but i dont want to run bundle exec everytime lol its alrdy long enough and it makes me unable to use my extension 😅

https://github.com/scarpe-team/scarpe/pull/324#issuecomment-1641709784

### Checklist

- [ ] Run tests locally
- [ ] Run linter(check for linter errors)

### CODE REVIEW

**Comment by gintama91** [comment]
on the same note can i update readme with vs code extension?

File: exe/scarpe [hunk 1]
```diff
@@ -55,6 +55,13 @@ end
 use_dev = ARGV.delete("--dev") ? true : false
 use_debug = ARGV.delete("--debug") ? true : false
 
+if use_dev or use_debug
```
Comment (noahgibbs) [inline exe/scarpe:58]: I would probably do this on use_dev but *not* use_debug.

Use_dev is when you're doing it locally from a repo. But use_debug is perfectly valid even if you're using Shoes outside any cloned Scarpe repo, just in your app somewhere.

Comment (noahgibbs) [reply exe/scarpe:58]: I'm not sure how we're gonna do the Rails-style dir-full-of-app stuff, but right now a lot of Shoes apps don't *use* a Gemfile.

**Comment by noahgibbs** [comment]
Yup, this is basically what I meant.

Yup, it'd be great to mention the VSCode extension in the README.

**Comment by gintama91** [comment]
removed debug and updated

**Comment by noahgibbs** [comment]
Huh. Why did I think Schwad had filed this? You did. Sorry, think I mentioned that wrong in a comment somewhere.

**Review by noahgibbs — APPROVED** [review]

**Pull request merged**